requirements/edx/base.txt

```diff
@@ -55,8 +54,9 @@ Markdown==2.2.1
 --allow-unverified meliae
 meliae==0.4.0
 mongoengine==0.7.10
+MySQL-python==1.2.5
 networkx==1.7
-nltk==2.0.4
+nltk>=2.0.4 # upgrade to 2.0.5 once edx-ora2 is compatible
```

Comment:
Sorry, I meant 2.0.5 (I edited my answer). I totally understand if you prefer to have pinned dependencies. You will just have to remember to upgrade both dependencies from edx-ora2 and edx-platform at the same time next time you upgrade :-)